### PR TITLE
Fix navigation issue

### PR DESCRIPTION
**Pull request by jassalrichy**

### CODE REVIEW

File: app/controllers/register/suitability/BeforeYouContinueController.scala [hunk 1]
```diff
@@ -50,12 +51,14 @@ class BeforeYouContinueController @Inject()(
...
-            case (Some(false), true, _)  => Ok(nonTaxableAgentView(draftId))
-            case (_, _, _) => Redirect(controllers.register.routes.SessionExpiredController.onPageLoad())
+        val trustHasUtr = request.userAnswers.get(TrustHaveAUTRPage)
+        val trustTaxable = request.userAnswers.get(TrustTaxableYesNoPage)
+        (trustHasUtr, trustTaxable, isAgent, is5mldEnabled) match {
+            case (Some(false), Some(true), _, _) => Ok(view(draftId))
+            case (Some(true), Some(true), _, _) => Ok(taxableView(draftId))
+            case (_, Some(false), false, _) => Ok(nonTaxableView(draftId))
+            case (_, Some(false), true, _)  => Ok(nonTaxableAgentView(draftId))
+            case (_, _, _, _) => Redirect(controllers.register.routes.SessionExpiredController.onPageLoad())
```
Comment (jackheslop96) [inline app/controllers/register/suitability/BeforeYouContinueController.scala:61]: Value of is5mldEnabled looks to be irrelevant in the pattern match?

Comment (jackheslop96) [inline app/controllers/register/suitability/BeforeYouContinueController.scala:61]: Can become `case _`

Comment (jassalrichy) [reply app/controllers/register/suitability/BeforeYouContinueController.scala]: Agree - Updated

Comment (jassalrichy) [reply app/controllers/register/suitability/BeforeYouContinueController.scala]: Updated

**Review by jackheslop96 — APPROVED** [review]
